integration-cli/docker_api_stats_test.go

```diff
@@ -82,14 +82,14 @@ func (s *DockerSuite) TestApiStatsStoppedContainerInGoroutines(c *check.C) {
 func (s *DockerSuite) TestApiStatsNetworkStats(c *check.C) {
 	testRequires(c, SameHostDaemon)
 	testRequires(c, DaemonIsLinux)
-	// Run container for 30 secs
+
 	out, _ := dockerCmd(c, "run", "-d", "busybox", "top")
```

Comment:
@wenchma I think you could even use the new `runSleepingContainer` function for these :angel: 
(https://github.com/docker/docker/blob/0f5aa19593dd2258a26adf37e41e00cbbbab8cf1/integration-cli/docker_utils.go#L1713)